packages/config/config/devices/0x012a/iq_outlet.json

```diff
@@ -0,0 +1,77 @@
+{
+	"manufacturer": "Qolsys",
+	"manufacturerId": "0x012a",
+	"label": "IQ Outlet",
+	"description": "Smart Plug-in Switch",
+	"devices": [
+		{
+			"productType": "0x4744",
+			"productId": "0x3033",
+			"zwaveAllianceId": 4133
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Set",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "3",
+			"label": "LED Indicator",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "On when load is off",
+					"value": 0
+				},
+				{
+					"label": "On when load is on",
+					"value": 1
+				},
+				{
+					"label": "Always Off",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Energy Monitoring Reporting Interval",
+			"description": "0 = disable energy monitoring, default 1 - 255 = reporting interval (minutes)",
```

Comment:
This doesn't conform with the style guide

Reply:
:-) I just missed deleting it, lol